### PR TITLE
Unapologetic Manhack / Viscerator Nerfs

### PR DESCRIPTION
**Pull request by Geevies**
1) Removes the flashing effect on manhack grenades. It lasted around two to three seconds, making escape nigh impossible.
2) Made manhacks do a random number between 10 and 15 damage.
3) Made manhacks easier to click on by giving them a small invisible square around their sprite.

image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Arrow768** [comment]
They are already pretty weak and can easily be destroyed.
Nerfing them just reduces their usefulness even further.

**Comment by Geevies** [comment]
@Arrow768 Should I put up a feedback thread on the forums? I think this change is well-liked among the people I've interviewed. Unless we can do an in-game poll?

**Comment by Geevies** [comment]
Side-note I forgot to mention. They only cost 3 TC, which means, if you're insane, you can buy 8 of these manhack grenades with a single TC to spare. This is equal to a whopping 40 manhacks. Due to them having 0 density, they can go into eachother's turfs and wreak havoc. I think bringing it in-line to its TC cost is fair.

**Comment by OneOneThreeEight** [comment]
Give it the flashing effect back.

File: code/game/objects/items/weapons/grenades/spawnergrenade.dm [hunk 1]
```diff
@@ -16,9 +17,12 @@
 		// Make a quick flash
 		var/turf/T = get_turf(src)
 		playsound(T, 'sound/effects/phasein.ogg', 100, 1)
-		for(var/mob/living/carbon/human/M in viewers(T, null))
-			if(M.eyecheck(TRUE) < FLASH_PROTECTION_MODERATE)
-				flick("e_flash", M.flash)
+		if (flash == 1)
```
Comment (skull132) [inline code/game/objects/items/weapons/grenades/spawnergrenade.dm]:
```suggestion
		if (flash)
```

File: code/game/objects/items/weapons/grenades/spawnergrenade.dm [hunk 2]
```diff
@@ -5,6 +5,7 @@
 	icon_state = "delivery"
 	item_state = "flashbang"
 	origin_tech = list(TECH_MATERIAL = 3, TECH_MAGNET = 4)
+	var/flash = 1 //this var handles whether we blind people when we explode
```
Comment (skull132) [inline code/game/objects/items/weapons/grenades/spawnergrenade.dm]:
```suggestion
	var/flash = TRUE // this var handles whether we blind people when we explode
```

File: code/game/objects/items/weapons/grenades/spawnergrenade.dm [hunk 3]
```diff
@@ -39,8 +43,9 @@
 	name = "manhack delivery grenade"
 	desc = "It is set to detonate in 5 seconds. It will unleash a swarm of deadly manhack robots that will attack everyone but you and your allies."
 	spawner_type = /mob/living/simple_animal/hostile/viscerator
-	deliveryamt = 5
+	deliveryamt = 5 //Five seems a bit much, but we'll keep it as-is.
 	origin_tech = list(TECH_MATERIAL = 3, TECH_MAGNET = 4, TECH_ILLEGAL = 4)
+	flash = 0
```
Comment (skull132) [inline code/game/objects/items/weapons/grenades/spawnergrenade.dm]:
```suggestion
	flash = FALSE
```

**Pull request merged**